### PR TITLE
model serving docs

### PR DESCRIPTION
**Pull request by aowen14**
Adding Docs for Model serving. 

Uses the example from the exampels repo for using VLLM to serve Mistral-7b. Also can be a starting point for showing how to serve TGI, VLLM, Cog, Ollama, Exllama, SGLang, and anything else that people would want to serve

### CODE REVIEW

File: modelserving.mdx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+#request the best instance that is available
+response = requests.request("POST", create_url, json=payload, headers=headers)
+#easy way to visually see if this request worked
+print(response.text)
+```
+
+Once we request it, Shadeform will provision the machine, and deploy a docker container based on the image, arguments, and environment variables that we selected. 
+
+We can see that this will deploy an openai compatible server with VLLM serving Mistral-7b-v0.1.
```
Comment (cholical) [inline modelserving.mdx]: nit: Can we make the v in vLLM lowercase? That's their preferred way of writing it

File: modelserving.mdx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  }
+}
+
+#request the best instance that is available
+response = requests.request("POST", create_url, json=payload, headers=headers)
+#easy way to visually see if this request worked
+print(response.text)
+```
+
+Once we request it, Shadeform will provision the machine, and deploy a docker container based on the image, arguments, and environment variables that we selected.
```
Comment (cholical) [inline modelserving.mdx:58]: Can we add that this step may take 5-10 minutes depending on the machine chosen and the size of the model weights

Comment (cholical) [reply modelserving.mdx:58]: Can we also add a link to the API reference - something along the lines of "for more information on the API fields, see <API Reference link>

Comment (aowen14) [reply modelserving.mdx:58]: Handled

File: modelserving.mdx (hunk 1 above)
Comment (aowen14) [reply modelserving.mdx]: Handled

**Pull request merged**